### PR TITLE
feat(ci): add CI workflow for examples

### PR DESCRIPTION
**Pull request by fiam**

### CODE REVIEW

File: scripts/test-example.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+
+usage()
```
Comment (StarpTech) [inline scripts/test-example.sh:3]: Awesome work! But we can drastically simplify it by using https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations

In that way, we can have one workflow that runs all examples in parallel with propert action integration (status, logging....)

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: scripts/test-example.sh (hunk 1 above)
Comment (StarpTech) [reply scripts/test-example.sh]: Why you can't use one job per example? In that way, you don't need to set up stuff that only one example required of 5. We can have up to 256 parallel jobs :smile:

Comment (StarpTech) [reply scripts/test-example.sh]: Done

**Review by YuriBuerov — APPROVED** [review]

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**